var/spack/repos/builtin/packages/agrep/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Agrep(MakefilePackage):
+    """AGREP - an approximate GREP.
+    Fast searching files for a string or regular expression,
+    with approximate matching capabilities and user-definable records.
+    Developed 1989-1991 by Udi Manber, Sun Wu et al. at the University
+    of Arizona."""
+
+    homepage = "http://www.tgries.de/agrep"
+    url      = "https://www.tgries.de/agrep/agrep-3.41.tgz"
+
+    version('3.41', sha256='0508eafaf9725fc67cc955eb6d32ba4f50138443a4fea4275508d2c3f67a234e')
+
+    def install(self, spec, prefix):
+        install('agrep', prefix)
```

Comment:
I would install this to prefix.bin so it shows up in PATH

Reply:
I refined it in this patch: https://github.com/spack/spack/pull/15165/commits/b59df9770cace151a8ae459f3523e9113bde91dc